idea_town/frontend/static-src/app/views/footer.js

```diff
@@ -0,0 +1,6 @@
+import BaseView from './base-view';
+import template from '../templates/footer';
```

Comment:
Was this file maybe included by mistake? There's no `templates/footer.js` and only `footer-view.js` gets used in `page-view.js`.

Gulp build on my machine breaks unless I delete this. (Also weird that Travis didn't complain)